src/hs27.jl

```diff
@@ -0,0 +1,36 @@
+# Hock and Schittkowski problem number 27.
+#
+#   Source:
+#   Problem 27 in
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lectures Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag, Heidelberg, 1981.
+#
+#   classification QR2-AN-3-1
```

Comment:
This should be `SQR2-AN-3-1`. Would you mind correcting it in a new pull request? Sorry I didn't notice it earlier. Thanks!